File: Project/Web-SPA/src/app/entities/rent-a-car-service.ts
import { Destination } from './destination';
import { Address } from './address';

export class RentACarService {

    id: number;
    adminId: number;
    name: string;
    address: Address;
    promoDescription: string;
    branches: Array<string>;
    averageRating: number;
    rates: Array<number>;

    constructor(name: string, address: Address) {
        this.name = name;
        this.address = address;
        this.promoDescription = '';
        this.branches = new Array<string>();
        this.averageRating = 0.0;
        this.rates = new Array<number>();
    }
}

File: Project/WebApi/WebApi/Models/RentACarService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApi.Models
{
    public class RentACarService
    {
        public int RentACarServiceId { get; set; }
        public string Name { get; set; }
        public string AdminId { get; set; }
        public RentACarServiceAdmin Admin { get; set; }
        public Address2 Address { get; set; }
        public string About { get; set; }
        public ICollection<Branch> Branches { get; set; }
        public ICollection<Car> Cars { get; set; }
        public byte[] LogoUrl { get; set; }
        public ICollection<RentCarServiceRates> Rates { get; set; }

        public RentACarService()
        {
            Branches = new HashSet<Branch>();
            Cars = new HashSet<Car>();
            Rates = new HashSet<RentCarServiceRates>();
        }
    }
}
